### PR TITLE
CORE-5084 Rewrite to add ARM support

### PR DESCRIPTION
**Pull request by davidcurrie**
Move away from Bitnami Helm charts to enable support for ARM. Provide minimal overrides to reinforce that this chart is only intended for development purposes. Uses KRaft for Kafka to minimise the number of running processes. The downside of this is that SCRAM is not supported so we use PLAIN for authentication. The chart NOTES provide information on the overrides that should be used with a subsequent Corda install, as well as details of how to connect to PostgreSQL and Kafka from outside of the Kubernetes cluster.

### CODE REVIEW

File: charts/corda-dev/kafka-config/server.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
+listeners=SASL_SSL://:9092,CONTROLLER://:9093,SASL_PLAINTEXT://:9094
```
Comment (JasonSinclair95) [inline charts/corda-dev/kafka-config/server.properties:1]: Why is this file needed?

File: charts/corda-dev/kafka-config/server.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+advertised.listeners=SASL_SSL://{{ include "corda-dev.kafkaName" . }}:9092,SASL_PLAINTEXT://localhost:9094
+inter.broker.listener.name=SASL_SSL
```
Comment (JasonSinclair95) [inline charts/corda-dev/kafka-config/server.properties:3]: What happens if sasl is not enabled? but your left with tls?

**Review by JasonSinclair95 — DISMISSED** [review]
LGTM

File: charts/corda-dev/kafka-config/server.properties (hunk 2 above)
Comment (davidcurrie) [reply charts/corda-dev/kafka-config/server.properties:3]: Unlike the original chart, there aren't options to disable TLS or auth - you always get both. That's the sort of thing I was hoping to solicit feedback on but, having had none, I'm going to keep it as simple as possible!

Comment (JasonSinclair95) [reply charts/corda-dev/kafka-config/server.properties:3]: ssl.key password, can we avoid hardcoding this?

File: charts/corda-dev/kafka-config/server.properties (hunk 1 above)
Comment (davidcurrie) [reply charts/corda-dev/kafka-config/server.properties:1]: I started out just specifying command line overrides for the Kafka options that I wanted to change but ended up overriding nearly all of them so it was just simpler to provide the entire config file to Kafka.

File: charts/corda-dev/kafka-config/server.properties (hunk 2 above)
Comment (davidcurrie) [reply charts/corda-dev/kafka-config/server.properties:3]: To be honest, I would have generated the keystore without a passphrase at all (given that both the keystore and password only exist within the container so if you have access to one, you have access to the other) but Kafka didn't seem to like not having one.

Comment (davidcurrie) [reply charts/corda-dev/kafka-config/server.properties:3]: Pushed an update to generate the keystore password.
